taf/src/main/java/pages/YandexDiskAccountPage.java

```diff
@@ -0,0 +1,230 @@
+        )).click();
+        new WebDriverWait(webDriver, 20)
+                .until(ExpectedConditions.visibilityOfElementLocated(buttonDeleteFile)).click();
+        return this;
+    }
+
+    public YandexDiskAccountPage deleteFileCompletly(String fileName) {
+        navigateTrashItem();
+        webDriver.findElement(By.xpath(
+                String.format("//div[@class='listing__items']//*[contains(text(),'%1$s') or contains(@title,'%1$s')]", fileName)
+        )).click();
+        new WebDriverWait(webDriver, 20)
+                .until(ExpectedConditions.visibilityOfElementLocated(buttonDeleteFromTrash)).click();
+        return this;
+    }
+
+    private YandexDiskAccountPage switchChromeTabs(int tabNumber) {
+        ArrayList<String> chromeTabs = new ArrayList<>(webDriver.getWindowHandles());
+        webDriver.switchTo().window(chromeTabs.get(tabNumber));
+        return this;
+    }
+
+    public boolean checkMainMenuItems() {
+        navigateRecentItem();
+        boolean checkRecentURL = webDriver.getCurrentUrl().contains("/client/recent");
+        navigateFilesItem();
+        boolean checkFilesURL = webDriver.getCurrentUrl().contains("/client/disk");
+        navigatePhotoItem();
+        boolean checkPhotoURL = webDriver.getCurrentUrl().contains("/client/photo");
+        navigateSharedItem();
+        boolean checkSharedURL = (webDriver.getCurrentUrl().contains("/client/shared")
+                || webDriver.getCurrentUrl().contains("/client/published"));
+        navigateHistoryItem();
+        boolean checkHistoryURL = webDriver.getCurrentUrl().contains("/client/journal");
+        navigateArchiveItem();
+        boolean checkArchiveURL = webDriver.getCurrentUrl().contains("/client/mail");
+        navigateTrashItem();
+        boolean checkTrashURL = webDriver.getCurrentUrl().contains("/client/trash");
+
+        return (checkRecentURL && checkFilesURL && checkPhotoURL && checkSharedURL && checkHistoryURL
+                && checkArchiveURL && checkTrashURL);
+    }
+
+    public boolean checkFolderCreated(String folderName) {
+        navigateFilesItem();
+        return !new WebDriverWait(webDriver, 20).until(
+                ExpectedConditions.visibilityOfAllElementsLocatedBy(By.xpath(
+                        String.format("//div[@class='listing__items']//*[contains(text(),'%1$s') or contains(@title,'%1$s')]", folderName)
+                ))).isEmpty();
+    }
+
+    public boolean visitCreatedFolder(String folderName) {
+        goToFolder(folderName);
+        return !new WebDriverWait(webDriver, 20).until(
+                ExpectedConditions.visibilityOfAllElementsLocatedBy(By.xpath(
+                        String.format("//h1[@title='%1$s']", folderName))
+                )).isEmpty();
+    }
+
+    public boolean checkCreatedFileExistsAndHasProperContent(String folderName, String fileName, String inputText) {
```

Comment:
A big and smart method. Divide method on two or more if method has several functionalities